x-pack/plugins/graph/public/application.ts

```diff
@@ -9,34 +9,34 @@
 // They can stay even after NP cutover
 import angular from 'angular';
 import { i18nDirective, i18nFilter, I18nProvider } from '@kbn/i18n/angular';
-
+import '../../../../webpackShims/ace';
```

Comment:
It couldn't find the module `ace`, so I import `webpackShims/ace`.
@flash1293 can we use such import in NP?